### PR TITLE
Minor tweaks and hacks to the current build

### PR DESCRIPTION
**Pull request by MaxG2D**
See commits for more info. Please, have mercy, I'm not a programmer :)

### CODE REVIEW

File: ref_vk/shaders/ray.rgen [hunk 1]
```diff
@@ -535,7 +535,7 @@ void main() {
 					// Refraction
 					if (rand01() < payload_opaque.transmissiveness) {
 						throughput *= material.baseColor;
```
Comment (0x4E69676874466F78) [inline ref_vk/shaders/ray.rgen:537]: Ты не удалил эту строчку, в итоге ты не починил #60.
https://github.com/w23/xash3d-fwgs/pull/285/files#diff-6c13a93e0df24a15163750928ca9fdb9d0850074e5ac490f38066c5d63e96c10L537

**Comment by 0x4E69676874466F78** [comment]
@MaxG2D сравнивать надо без PBR текстуры (в `\valve\pbr\halflife.wad\materials.mat`), она вносит свои искажения.

File: ref_vk/shaders/ray.rgen (hunk 1 above)
Comment (MaxG2D) [reply ref_vk/shaders/ray.rgen]: Oops, it's fixed now with 655934e. Thanks!

**Comment by MaxG2D** [comment]
> @MaxG2D it is necessary to compare the texture without PBR (in `\valve\pbr\halflife.wad\materials.mat`), it introduces its own distortions. But in general, this hack should not be possible in principle if additive works correctly.




As you can see, there's still some weird colorshift, but it's miles better than just simply use base_color

**Comment by 0x4E69676874466F78** [comment]
@MaxG2D 
> But in general, this hack should not be possible in principle if additive works correctly.

No, the translation must be "hack shouldn't exist (no need) when additive works correctly".

> As you can see, there's still some weird colorshift, but it's miles better than just simply use base_color

Посмотри что происходит на test_gradient.

**Comment by 0x4E69676874466F78** [comment]
@MaxG2D да выглядит почти правильно, но как видишь где должен быть белый там не белый а серый, оттого и цвета тоже не совсем те.

**Comment by MaxG2D** [comment]
Added more fixes for issues #284 #230 #145

**Pull request closed without merging**